tailwindcss/deps.ts

```diff
@@ -0,0 +1,31 @@
+import type { Page } from "lume/core.ts";
+
+import dbin from "dbin/mod.ts";
```

Comment:
Import the full url of dbin `https://deno.land/x/dbin@v0.2.0/`. And it must be the version 0.2.0 (you're using the version 0.1.1 that doesn't work with the tailwind binary).